v5/merge.go

```diff
@@ -154,6 +162,12 @@ func doMergePatch(docData, patchData []byte, mergeMerge bool) ([]byte, error) {
 			patchErr = json.Unmarshal(patchData, patchAry)
 
 			if patchErr != nil {
+				// Per spec if patch is a string is should override the docData
+				isStr, err := regexp.Match(`".*"`, patchData)
```

Comment:
I think there must be a more lightweight way to do this test. What are you testing for? That the patchData contains only UTF-8 values?

Reply:
Sorry, this got lost of the updates.  I'm open to any updates, just wanted to match the spec correctly